tests/examplefiles/savi/example.savi

```diff
@@ -23,6 +21,6 @@
     integers Array(U64)'val = [99, 100, 101]
 
     // Check addition overflow for various pairs of addends.
-    @assert = try (add_overflow!(U64.max_value, 1), False | True)
-    @assert = try (add_overflow!(U64.max_value, 0), True | False)
-    @assert = try (add_overflow!(integers[0]!, 1), True | False)
```

Comment:
This makes `True` and `False` no longer tested. I think it's more important to test all lexer rules than to have an example that is well-written according to current language standards.

Reply:
Please note that `True` and `False` are highlighted according to the same rules as all other capitalized identifiers (such as `Main`, `Spec`, `U64`, and many other such identifiers in this example) - they are not special-case tokens as they are in some languages.

Hence, there is no decrease here in the amount of lexer rules being tested.

Reply:
Ah, sorry for not looking closely enough.